src/sentry/web/api.py

```diff
@@ -306,6 +309,9 @@ def post(self, request, **kwargs):
             # bubble up as an APIError.
             data = None
 
+        if ENABLE_PUBSUB:
```

Comment:
We might want to change this to `if ENABLE_PUBSUB and data is not None:` Since it's possible, given L313, that data is junk.

Reply:
done